emission/net/api/cfc_webapp.py

```diff
@@ -73,17 +73,17 @@
 #Simple path that serves up a static landing page with javascript in it
 @route('/')
 def index():
-  return static_file("server/index.html", static_path)
+  return static_file("server/home.html", static_path)
 
 # Bunch of static pages that constitute our website
 # Should we have gone for something like django instead after all?
 # If this gets to be too much, we should definitely consider that
 @route("/<filename>")
 def doc(filename):
-  if filename != "privacy" and filename != "support" and filename != "about" and filename != "consent":
+  if filename != "privacy" and filename != "home" and filename != "about" and filename != "index":
     return HTTPError(404, "Don't try to hack me, you evil spammer")
   else:
-    return static_file("%s.html" % filename, "%s/docs/" % static_path)
+    return static_file("%s.html" % filename, "%s/server/" % static_path)
```

Comment:
Did you check to see whether the existing
http://localhost:8080/privacy
and
http://localhost:8080/consent
URLs still work after this change?

Those are linked from the app store pages, and the IRB consent document. So we should ensure that they keep working.

If you have all the assets routes below, do you still need these?